techniques/general/G56.html

```diff
@@ -52,111 +49,105 @@
       </section>
       <section class="example" id="example-3-insufficient-audio-contrast-between-a-narrator-and-background-music">
-         <h3>Example 3: Insufficient Audio Contrast between a narrator and background music</h3>
+         <h3>事例 3: ナレーターの音声と背景音楽との不十分な音声コントラスト</h3>
          <section id="audio-example-of-the-failure">
-            <h4>Audio example of the failure</h4>
-            <p>This example demonstrates a voice with music in the background in which the voice is not 20 DB above the background. The voice (foreground) is at -18 decibels and the music (background) is at about -16 decibels making the foreground only 2 decibels louder than the background.</p>
+            <h4>失敗例の音声サンプル</h4>
+            <p>この事例では、ナレーターの音声と背景音楽があり、主音声は背景音楽よりも 20 デシベル以上は大きくない。主音声 (前景) は -18 デシベル、音楽 (背景) は -16 デシベルで録音しているため、前景は背景よりも 2 デシベルしか大きくない。</p>
             <p class="working-example">
-                <a href="../../working-examples/audio-contrast/bad-contrast.mp3">Audio Example: Foreground is less than 20 decibels above the background (mp3)</a>
+                <a href="https://www.w3.org/WAI/WCAG22/working-examples/audio-contrast/bad-contrast.mp3">音声サンプル: 前景音声と背景音楽の差が 20 デシベル未満である (mp3 形式)</a>
               </p>
           </section>
           <section id="transcript-of-audio-example-bad-contrast">
-            <h4>Transcript of audio example (bad contrast)</h4>
+            <h4>音声サンプル (不十分なコントラスト) のトランスクリプト</h4>
             <p>"This is an example of a voice that is not loud enough against the background. The voice which is the foreground is only about 2 decibels above the background. Therefore is difficult to understand for a person who is hard of hearing. It is hard to discern one word from the next. This is an example of what not to do."</p>
           </section>
           <section id="visual-example-of-the-failure">
-            <h4>Visual example of the failure</h4>
-            <p>The highlighted section contains foreground and background. The wave is almost the same size the section that contains only background, which means the background is too loud in comparison to the foreground voice.</p>
+            <h4>失敗例の音声サンプルを視覚的に示したもの</h4>
+            <p>強調表示にしたセクションは、前景音声と背景音楽を含んだ部分である。背景音楽しかないセクションと波形がほぼ同じ大きさであり、前景音声に比べて背景音楽が大きすぎるということである。</p>
             <figure id="figure-visual-representation-poor-contrast">
-            	<img alt="A display showing a sound wave. A short section on the left is just background noise with a range of -35db to +35db. The main section of the wave has background and foreground noise, with a range of almost -70db to +70db. The final background-only section has a spike of a similar size." src="img/Audio_4.jpg">
-            	<figcaption>Visual representation of bad contrast.</figcaption>
+            	<img alt="音波を表示するディスプレイ。左側の短いセクションは、-35db から +35db の範囲の背景ノイズのみで構成されている。波形の主要セクションには、背景ノイズと前景ノイズが含まれており、範囲はほぼ-70db から +70db である。最後の背景のみのセクションには、同様の大きさのスパイクがある。" src="img/Audio_4.jpg">
```

Comment:
ここも、前景はノイズではないはずなのですが、原文が「foreground noise」となってしまっていますね...。
一応、修正提案です。
```suggestion
            	<img alt="音波を表示するディスプレイ。左側の短いセクションは、-35db から +35db の範囲の背景ノイズのみで構成されている。波形の主要セクションには、背景ノイズと前景音が含まれており、範囲はほぼ-70db から +70db である。最後の背景のみのセクションには、同様の大きさのスパイクがある。" src="img/Audio_4.jpg">
```